chainio/clients/fireblocks/contract_call.go

```diff
@@ -34,7 +34,7 @@ type extraParams struct {
 
 type ContractCallRequest struct {
 	Operation       TransactionOperation `json:"operation"`
-	ExternalTxID    string               `json:"externalTxId"`
+	ExternalTxID    string               `json:"externalTxId,omitempty"`
```

Comment:
add the explanation you made in the PR definition here as a comment, so we know why omitempty is fine

Reply:
👍 clarified that this is an optional field